package/Makefile

```diff
@@ -68,23 +85,32 @@ help:
 	@echo "   Here is a summary of the available targets:"
 	@echo
 	@echo "   [Packaging]"
-	@echo "     package        - Build and package the OpenPilot distributable"
-	@echo "     package_flight - Build and package the OpenPilot flight firmware only"
+	@echo "     package        			- Build and package the Tau Labs distributable (DEB)"
```

Comment:
Same mistake as on the top level makefile, installer is now built with "package_installer" including dmg on osx